server/README.md

```
@@ -54,6 +54,24 @@ APP_NAME=cores pnpm run start:dev
+- server 폴더에 .env.development 파일을 생성합니다.
+- .env.development 파일 내에 다음과 같은 내용을 추가합니다.
+```
+MYSQL_HOST=local # 호스트 명에 따라 이름을 변경해줍니다.
+MYSQL_DATABASE=job-play-dev
+MYSQL_ROOT_USER=root
+MYSQL_ROOT_PASSWORD=root
+MYSQL_PORT=3306 # 기본 포트 3306
+```
+Docker mysql 이미지를 다운 받습니다.
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)